---
float64 nitrate_val
string flag
